ambry-account/src/main/java/com/github/ambry/account/AbstractAccountService.java

```diff
@@ -269,6 +269,17 @@ public boolean removeAccountUpdateConsumer(Consumer<Collection<Account>> account
     return accountUpdateConsumers.remove(accountUpdateConsumer);
   }
 
+  @Override
+  public void addDataset(short accountId, short containerId, Dataset dataset) throws AccountServiceException {
+    throw new UnsupportedOperationException("This method is not supported");
```

Comment:
the Interface AccountService already has default implementation, we don't need it here.

Reply:
Removed